config/test.exs

```diff
@@ -7,9 +7,35 @@ config :prima_auth0_ex,
   token_cache: TokenCacheMock,
   token_service: TokenServiceMock
 
+config :prima_auth0_ex, :redis,
+  enabled: true,
+  encryption_key: "uhOrqKvUi9gHnmwr60P2E1hiCSD2dtXK1i6dqkU4RTA=",
+  connection_uri: "redis://redis:6379",
+  ssl_enabled: false,
+  ssl_allow_wildcard_certificates: false
+
 config :prima_auth0_ex, :client,
```

Comment:
Since we're forcing people to update their configs no matter what wouldn't it make sense to remove special handling for the default client's config and force people to use `default_client`?

Reply:
For the "remove special handling for the default client" I kinda agree, I was leaving it because it may be strange for people having to define a `:clients` even though they have a single client implementation

For the "use `default_client`", what do you mean? If we define a `:default_client` under `:clients` it wouldn't be treated any differently from other clients

Reply:
I mean still have a default argument `:default_client`, so when updating people who only use a single client can just create a named client called `:default_client` and not have to think about it any more than that

Reply:
Aaaah I see, you are right, they would need to always specify a client...! duh

I'll see what I can do

Reply:
Refactored! Now `:client` is gone